spec/features/user/user_dashboard_spec.rb

```diff
@@ -70,6 +70,7 @@
       expect(page).to have_link('futbol')
       expect(page).to have_link('battleship')
       expect(page).to have_link('1911Enigma')
+      # expect(page).to have_css('.repo', count: 5)
```

Comment:
Cool. We'll save this for later when we go back to clean up the tests and the rest of the suite.